mlflow/utils/credentials.py

```diff
@@ -41,3 +46,129 @@ def read_mlflow_creds() -> MlflowCreds:
         username=username_env or username_file,
         password=password_env or password_file,
     )
+
+
+def login(backend="databricks"):
+    """Configure MLflow server authentication and connect MLflow to tracking server.
+
+    This method provides a simple way to connect MLflow to its tracking server. Currently only
+    Databricks tracking server is supported. Users will be prompted to enter the credentials if no
+    existing Databricks profile is found, and the credentials will be saved to `~/.databrickscfg`.
+
+    Args:
+        backend: string, the backend of the tracking server. Currently only "databricks" is
+            supported.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        mlflow.login()
+        with mlflow.start_run():
+            mlflow.log_param("p", 0)
+    """
+    if backend == "databricks":
+        _databricks_login()
+    else:
+        raise ValueError(
```

Comment:
Can we raise an MLflowException with the invalid argument exception to be more precise?